### PR TITLE
Clear redirects formating

### PR DESCRIPTION
**Pull request by c-schwarze**

### CODE REVIEW

File: tinker/templates/admin/redirects/clear-redirects.html [hunk 1]
```diff
@@ -1,40 +1,21 @@
+<div>
 {% if changed|length == 0 and deleted|length == 0 %}
-    <p>There are no changes or deletions to worry about. Good work!</p>
+    There are no changes or deletions to worry about. Good work!<br>
```
Comment (phg49389) [inline tinker/templates/admin/redirects/clear-redirects.html]: Nitpick: `<br/>`

Comment (c-schwarze) [reply tinker/templates/admin/redirects/clear-redirects.html]: Yeah I removed those because they didn't work in the test 🤷‍♂️

Comment (phg49389) [reply tinker/templates/admin/redirects/clear-redirects.html]: Yeah, but it should be `<br/>` not `<br>`. That's the nitpick

Comment (c-schwarze) [reply tinker/templates/admin/redirects/clear-redirects.html]: Somehow going through: html template > email > FP case creation, <br/> didn't carry over, but <br> did.

Comment (phg49389) [reply tinker/templates/admin/redirects/clear-redirects.html]: Rip

**Pull request merged**